test/services/pilot/catalog-credential-mock/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/ibm/the-mesh-for-data/test/services/pilot/catalog-credential-mock
+
+require github.com/ibm/the-mesh-for-data v0.0.0
+
+replace github.com/ibm/the-mesh-for-data v0.0.0 => ../../../..
+
+replace github.com/ibm/the-mesh-for-data/connectors/vault v0.0.0 => ../../../../connectors/vault
+
+go 1.13
+
+require (
+	github.com/golang/protobuf v1.4.2
+	github.com/hashicorp/vault/api v1.0.4
+	github.com/ibm/the-mesh-for-data/connectors/vault v0.0.0
+	google.golang.org/grpc v1.28.1
+)
```

Comment:
please remove, see how @froesef builds everything on the host so as to shorted build time by using Travis cache. Once build otside there is no need to maintain a prove go.mod

Reply:
fixed by removing the private  go.mod files from catalog-crendential-mock/ folder and also from connectors/vault and connectors/opa folders. Thanks.